kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -163,6 +163,10 @@ data:
     #   tail_buf_maxsize_megabytes = "1"              # defautl value is 32kb (comment out this line for default)
     #   tail_ignore_older = "5m"                      # default value same as fluent-bit default i.e.0m
 
+    # Controls max event rate for mdsd. Increasing this also increases memory consumption
```

Comment:
I think, we can remove the documentation for now and we can add later if required based on the feedback from high scale syslog + container log scenario.